ts/Stock/StockToolsBindings.ts

```diff
@@ -19,11 +19,13 @@ import type Point from '../Core/Series/Point';
 import H from '../Core/Globals.js';
 import NavigationBindings from '../Extensions/Annotations/NavigationBindings.js';
 import U from '../Core/Utilities.js';
+import dataGrouping from '../Extensions/DataGrouping';
```

Comment:
This needs to have the file extension, or otherwise it will fail as an ES6 module.

Reply:
I'm surprised it's even here. Already made a commit that removes that line, probably IDE auto-imported this during development.. thanks!

Reply:
Done 👍